### PR TITLE
doc: Added working with PMICs section

### PR DESCRIPTION
**Pull request by aasinclair**
Added Working with PMICs section to top level
Added nPM1300 features to new section

### CODE REVIEW

File: doc/nrf/working_with_pmic/npm1300/features.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+.. contents::
+    :local:
+    :depth: 2
+
+Introduction
+************
+
+nPM1300 is a Power Management IC (PMIC) with a linear-mode lithium-ion, lithium-polymer, and lithium ferro-phosphate battery charger.
+It has two dual-mode buck regulators, two dual purpose LDO/load switches, 3 LED drivers and 5 GPIOs.
```
Comment (peknis) [inline doc/nrf/working_with_pmic/npm1300/features.rst:14]:
```suggestion
It has two dual-mode buck regulators, two dual purpose LDO/load switches, three LED drivers and five GPIOs.
```

File: doc/nrf/working_with_pmic/npm1300/features.rst [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+nPM1300 is the perfect companion for nRF52, nRF53 series SoCs, and the nRF91 series SiP.
```
Comment (peknis) [inline doc/nrf/working_with_pmic/npm1300/features.rst:16]:
```suggestion
nPM1300 is the perfect companion for nRF52 and nRF53 Series SoCs, and the nRF91 series SiP.
```

File: doc/nrf/working_with_pmic/npm1300/features.rst [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+It is ideal for compact and advanced IoT products such as advanced wearables and portable medical applications.
```
Comment (peknis) [inline doc/nrf/working_with_pmic/npm1300/features.rst:17]:
```suggestion
It is ideal for compact and advanced IoT products such as wearables and portable medical applications.
```

File: doc/nrf/working_with_pmic/npm1300/gs.rst [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+###############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This section gets you started with your nPM1300 :term:`Evaluation Kit (EK)` using the |NCS|.
+It tells you how to build and flash the :ref:`npm1300_fuel_gauge` sample and perform a quick test of your EK.
+
+If this is your first time developing with a Nordic DK, you should follow the appropriate getting started tutorial first:
```
Comment (peknis) [inline doc/nrf/working_with_pmic/npm1300/gs.rst:13]:
```suggestion
If this is your first time developing with a Nordic DK, read the appropriate getting started tutorial first:
```

File: doc/nrf/working_with_pmic/npm1300/gs.rst [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+Install `nRF Connect for Desktop`_.
+After installing and starting the application, install the Programmer app.
+
+.. _npm1300_gs_building:
+
+Building and programming the sample
+***********************************
+
+Follow the detailed instructions in the :ref:`npm1300_fuel_gauge` sample to build the sample and flash it to the DK.
```
Comment (peknis) [inline doc/nrf/working_with_pmic/npm1300/gs.rst:64]:
```suggestion
Follow the detailed instructions in the :ref:`npm1300_fuel_gauge` sample documentation to build the sample and flash it to the DK.
```

**Review by peknis — CHANGES_REQUESTED** [review]

File: doc/nrf/working_with_pmic/npm1300/gs.rst [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+   PMIC device ok
+   V: 4.101, I: 0.000, T: 23.06, SoC: 93.09, TTE: nan, TTF: nan
+
+Next steps
+**********
+
+You have now completed getting started with the nPM1300 EK.
+See the following links for where to go next:
+
+* The EK `User Guide <nPM1300 EK User Guide_>`_ for detailed information related to the nPM1300 EK.
```
Comment (WalkingTalkingPotato) [inline doc/nrf/working_with_pmic/npm1300/gs.rst:84]: Page not available yet.

**Review by WalkingTalkingPotato — APPROVED** [review]

File: doc/nrf/releases/release-notes-changelog.rst [hunk 7]
```diff
@@ -542,3 +542,4 @@ Added:
 
 * A page on :ref:`ug_wireless_coexistence` in :ref:`protocols`.
 * Pages on :ref:`thread_device_types` and :ref:`thread_sed_ssed` to the :ref:`ug_thread` documentation.
+* A new section "Working with PMICs", containing :ref:`ug_npm1300_features` and :ref:`ug_npm1300_gs`.
```
Comment (peknis) [inline doc/nrf/releases/release-notes-changelog.rst:545]:
```suggestion
* A new section :ref:`ug_pmic`, containing :ref:`ug_npm1300_features` and :ref:`ug_npm1300_gs`.
```

**Review by peknis — CHANGES_REQUESTED** [review]

File: doc/nrf/links.txt [hunk 8]
```diff
@@ -377,6 +377,9 @@
 
 .. _`nRF21540 DK product page`: https://www.nordicsemi.com/Products/Low-power-short-range-wireless/nRF21540
 
+.. _`nPM1300 product website`: https://www.nordicsemi.com/products/npm1300
+.. _`nPM1300 EK product page`: https://www.nordicsemi.com/Products/Development-hardware/nPM1300-EK
```
[the remark this answers is not in the document]
Comment (aasinclair) [reply doc/nrf/links.txt:396]: Linked pages will go live today

**Review by peknis — APPROVED** [review]

**Review by b-gent — APPROVED** [review]

**Pull request merged**